python/py-pytorch/Portfile

```diff
@@ -4,10 +4,11 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           mpi 1.0
 PortGroup           python 1.0
+PortGroup           xcodeversion 1.0
```

Comment:
@catap You can remove the xcodeversion portgroup since you're not using it. (I see you are using the `${xcodeversion}` variable below, but that variable is provided by MacPorts base, not by the xcodeversion portgroup.)

Reply:
> @catap You can remove the xcodeversion portgroup since you're not using it. (I see you are using the `${xcodeversion}` variable below, but that variable is provided by MacPorts base, not by the xcodeversion portgroup.)

Great catch Ryan, I'll add that to my to-do list!